ktor-client/ktor-client-android/jvm/src/io/ktor/client/engine/android/AndroidClientEngine.kt

```diff
@@ -27,15 +27,15 @@ class AndroidClientEngine(override val config: AndroidEngineConfig) : HttpClient
     override suspend fun execute(
         data: HttpRequestData
     ): HttpResponseData {
-        val callContext = createCallContext()
+        val callContext: CoroutineContext = createCallContext()
         return async(callContext) {
-            val requestTime = GMTDate()
+            val requestTime: GMTDate = GMTDate()
```

Comment:
Why so many explicit types?

Reply:
It was slow to edit code with dev plugin :(